src/main/kotlin/uk/gov/justice/digital/hmpps/courthearingeventreceiver/model/Contact.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.courthearingeventreceiver.model
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class Contact(
+
+  @JsonProperty("home")
+  val home: String,
```

Comment:
These will need to be optional (`String?`) or it will fail if some are present but not others

Reply:
Yes, my bad. Updated